samples/bidask-to-ohlc/bidask-to-ohlc.py

```diff
@@ -57,7 +57,7 @@ def runstrat():
         timeframe=bt.TimeFrame.Ticks)
 
     cerebro.resampledata(data,
-                         timeframe=bt.TimeFrame.Ticks,
```

Comment:
The purpose of the sample is to resample to OHLC and not to make it Minutes based.

Reply:
It might not necessarily need to be Minutes. but if you don't fill the right timeframe, the sample doesn't work (hence the fix). 
Here is the original output:
```
2016-03-01 23:59:59.999989,130.52,130.52,130.52,130.52,3000000.0
2016-03-01 23:59:59.999989,130.51,130.53,130.53,130.53,2000000.0
2016-03-01 23:59:59.999989,130.54,130.55,130.55,130.55,1650000.0
2016-03-01 23:59:59.999989,130.5,130.5,130.5,130.495,2300000.0
2016-03-01 23:59:59.999989,130.48,130.48,130.48,130.47,1500000.0

```

Would you mind to interpret this?
The OHLC doesn't seem to work. (it seems like the day timeframe is used and the high and low not displayed correctly).

Reply:
It might not necessarily need to be Minutes. but if you don't fill the right timeframe, the sample doesn't work (hence the fix).
Here is the original output:
```
2016-03-01 23:59:59.999989,130.52,130.52,130.52,130.52,3000000.0
2016-03-01 23:59:59.999989,130.51,130.53,130.53,130.53,2000000.0
2016-03-01 23:59:59.999989,130.54,130.55,130.55,130.55,1650000.0
2016-03-01 23:59:59.999989,130.5,130.5,130.5,130.495,2300000.0
2016-03-01 23:59:59.999989,130.48,130.48,130.48,130.47,1500000.0
```

The OHLC doesn't seem to work. (it seems like the day timeframe is used and the high and low not displayed correctly).